docs/pages/other-env.mdx

```diff
@@ -0,0 +1,193 @@
+  "blockHash": "0xedb36756d11b591fb79b47be430b434b963abb2fb29a9c3766b505dd4f446333",
+  "blockNumber": "0xe",
+  "transactionHash": "0xf9ea723c30accef7b4c6b2bee9ab44c26d31ca40569d7c61f532ebc53312b249",
+  "transactionIndex": "0x0",
+  "logIndex": "0x0",
+  "transactionLogIndex": "0x0",
+  "removed": false
+}
+```
+
+The relevant fields are `address`, `topics`, and `data`.
+
+- `address` is the address of the `World` that emitted the event.
+- `topics` includes 32 byte values that are indexed by the blockchain.
+  By convention, the first topic is a hash of the event type.
+  For known event types (such as those in MUD), you can look for the original event type [in `4byte.directory`](https://www.4byte.directory/event-signatures/).
+- `data` is all the non indexed fields.
+  Fields are stored in 32 byte chunks.
+
+For our event, [you can see the event type here](https://www.4byte.directory/event-signatures/?bytes_signature=0x912af873e852235aae78a1d25ae9bb28b616a67c36898c53a14fd8184504ee32).
+
+```solidity
+StoreSetRecord(bytes32,bytes32[],bytes)
+```
+
+This is the begining of the data, divided into 32 byte words:
+
+| Word number | Data                                                             |
+| ----------: | ---------------------------------------------------------------- |
+|           0 | 6d756473746f726500000000000000005461626c657300000000000000000000 |
+|           1 | 0000000000000000000000000000000000000000000000000000000000000060 |
+|           2 | 00000000000000000000000000000000000000000000000000000000000000a0 |
+
+The first word is the table.
+It is two ASCII strings, one in byte 0 and one in byte 16.
```

Comment:
This is not necessarily the case, it is a convention as of now; but the MUD protocol supports any bytes32 table ID.